iothub/service/samples/how to guides/CleanupDevicesSample/CleanupDevicesSample.cs

```diff
@@ -193,7 +210,7 @@ private static Uri GetStorageAccountSasUriForCleanupJob(BlobContainerClient blob
                 | BlobContainerSasPermissions.Read
                 | BlobContainerSasPermissions.Delete;
 
-            var sasBuilder = new BlobSasBuilder(sasPermissions, DateTimeOffset.UtcNow.AddHours(1))
+            var sasBuilder = new BlobSasBuilder(sasPermissions, DateTimeOffset.UtcNow.AddHours(12))
```

Comment:
We monitor the job for up to 4 hours, but it'll die sooner if it loses access to storage.